gaffer-as-a-service/pom.xml

```diff
@@ -40,7 +40,6 @@
         <kubernetes.version>11.0.1</kubernetes.version>
         <fasterxml.jackson.version>2.11.2</fasterxml.jackson.version>
         <javax.xml.bind.version>2.3.0</javax.xml.bind.version>
-        <slf4j.api.version>1.7.25</slf4j.api.version>
```

Comment:
I thought you needed slf4j AND log4j dependencies?

Reply:
This is also creating a merge conflict

Reply:
Updated the log4j and slf4j dependencies and resolved merge conflict